### DISCUSSION TITLE
is air sdk  50.0.1.1 support android 32 bit devices?

### DISCUSSION THREAD

**Discussion by mehmury** [airsdk/Adobe-Runtime-Support#2204] [Q&A]
I needed to update an application with the air sdk version 50.0.1.1. The application does not work on my 32 bit samsung tablet during testing. It crashes after the splash screen.
Does this sdk support android 32 bit devices?

**Comment by ajwfrost** [comment]
It should do! Do you see anything in the logcat output, I'm wondering whether it's picking up the wrong binary either from the core AIR runtime or from an ANE...

Also just thinking about it .. what version of Android is that device running? I'm 99% sure we should be compatible down to sdk API 16 = 4.1 (Jellybean).

**Reply by mehmury** [reply]
The 32 bit android device I am using is an old Samsung tablet. It runs on Android 4.4.4.
The ANEs I use are distriqt Adverds Ane and IDFA ANE and dependencies anes.
Even if I clear all ANEs and compile only with AIR SDK 50.0.1.1, the application crashes on 32-bit android device.
[adt.log](https://github.com/airsdk/Adobe-Runtime-Support/files/9803122/adt.log)
I attached test log file results from my Animate usb device testing.
thanks

**Reply by ajwfrost** [reply]
Thanks, we've just done some checks too. Seems like this is the same with 50.0.0.1 as well so it's not the openssl update that's causing the problem. It's working on Android 5.0 (API 21) - will have to check whether this is by design, I don't recall us updating the minSdk setting. But it also appears to be the native library that's not loading properly, can't see why this should differ based on the API levels..
